### PR TITLE
Cookiecutter: fix test_install.py + test deps: unpin pytest-asyncio

### PR DESCRIPTION
**Pull request by masipcat**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -78,7 +78,7 @@
             "docker",
             "backoff",
             "psycopg2-binary",
-            "pytest-asyncio==0.10.0",
+            "pytest-asyncio",
```
Comment (masipcat) [inline setup.py:81]: So we can pin the version in the application depending on the pytest version (pytest 6.0.0 requires pytest-asyncio > 0.10.0)

Comment (vangheem) [reply setup.py:81]: Yes, it's really not a good idea to force a specific version.

**Comment by masipcat** [comment]
Can you do a release please?

**Comment by vangheem** [comment]
done!

**Comment by masipcat** [comment]
thank you :-)

**Pull request merged**